### PR TITLE
[DOCS] Compile tool docs change

### PR DESCRIPTION
**Pull request by tsavina**
### Details:
 - *item1*
 - *...*

### Tickets:
 - *ticket-id*

### CODE REVIEW

File: docs/Documentation/openvino_ecosystem.md [hunk 1]
```diff
@@ -104,5 +105,18 @@ More resources:
 * `PyPI <https://pypi.org/project/datumaro/>`__  
 * `GitHub <https://github.com/openvinotoolkit/datumaro>`__  
 
+Compile Tool 
+#############
+
+Compile tool is deprecated since the 2023.0 OpenVINO release. If you need to compile a model for inference on a specific device, use the following script: 
+
+.. code-block:: python
```
Comment (ilyachur) [inline docs/Documentation/openvino_ecosystem.md:113]: I propose to add code snippets and reuse it here: https://github.com/openvinotoolkit/openvino/tree/master/docs/snippets

Also please add the snippet for C++

Comment (tsavina) [reply docs/Documentation/openvino_ecosystem.md]: > I propose to add code snippets and reuse it here: https://github.com/openvinotoolkit/openvino/tree/master/docs/snippets
> 
> Also please add the snippet for C++

snippets added but I need help with the cpp content

File: docs/snippets/compile_model.py [hunk 2]
```diff
@@ -0,0 +1,5 @@
+#! [compile_model]
+
+ov.Core().compile_model(device, modelPath, properties).export_model(compiled_blob)
```
Comment (ilya-lavrenov) [inline docs/snippets/compile_model.py:3]: let's add import

File: docs/snippets/compile_model.cpp [hunk 3]
```diff
@@ -0,0 +1,8 @@
+
+{
```
Comment (ilya-lavrenov) [inline docs/snippets/compile_model.cpp:2]: let's add include path

**Comment by ilyachur** [comment]
You need to fix code snippets:
```
/home/runner/work/openvino/openvino/docs/snippets/compile_model.cpp:2:1: error: expected unqualified-id before ‘{’ token
    2 | {
      | ^
```

File: docs/snippets/compile_model.cpp [hunk 4]
```diff
@@ -0,0 +1,10 @@
...
+//! [compile_model]
+
+#include <openvino/runtime/core.hpp>
+
+ov::Core.compile_model(device, modelPath, properties).export_model(compiled_blob);
+
+//!  [compile_model]
+}
```
Comment (ilyachur) [inline docs/snippets/compile_model.cpp:10]:
```suggestion
//! [compile_model]

#include <openvino/runtime/core.hpp>

ov::Core.compile_model(device, modelPath, properties).export_model(compiled_blob);

//!  [compile_model]
```
Also propose to rename snippet name to `export_compiled_mode`

File: docs/snippets/export_compiled_model.cpp [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+std::stringstream stream;
+
+ov::CompiledModel model = core.compile_model("modelPath", "deviceName");
+
+model.export_model(stream);
+
+    return 0;
+}
+
+//!  [export_compiled_model]
```
Comment (ilyachur) [inline docs/snippets/export_compiled_model.cpp:18]:
```suggestion

#include <openvino/runtime/core.hpp>

int main() {

//! [export_compiled_model]
ov::Core core;

std::stringstream stream;

ov::CompiledModel model = core.compile_model("modelPath", "deviceName");

model.export_model(stream);

//!  [export_compiled_model]

return 0;
}

```

**Review by ilyachur — APPROVED** [review]

**Pull request merged**